cmake/CompilerFlags.cmake

```diff
@@ -23,10 +18,16 @@ set(GCC_DEBUG
 set(GCC_RELEASE -O3)  # Level 3 optimisation. Could also consider -fpack-derived
 
 # Intel 
+set(INTEL_BASE
+    -stand f08          # Fortran standard set to 2008
+    -implicitnone       # Give warning when undeclared variable is used
```

Comment:
Perhaps a more appropriate description is:
"Sets the default type of a variable to undefined"